graylog2-shared/src/main/java/org/graylog2/shared/stats/ThroughputStats.java

```diff
@@ -38,32 +37,24 @@
     public ThroughputStats() {
         this.currentThroughput = 0;
         this.throughputCounter = new Counter();
-        this.benchmarkCounter = new Counter();
-        this.streamThroughput = new AtomicReference<ConcurrentHashMap<String, Counter>>(new ConcurrentHashMap<String, Counter>());
-        this.currentStreamThroughput =  new AtomicReference<HashMap<String, Counter>>();
-
+        this.streamThroughput = new AtomicReference<>(new ConcurrentHashMap<String, Counter>());
+        this.currentStreamThroughput =  new AtomicReference<>();
     }
 
+    @Deprecated
     public long getCurrentThroughput() {
```

Comment:
We agreed offline that we will clean up the deprecated methods in a separate PR after radio is fixed, too.